index.html

```diff
@@ -315,6 +315,10 @@ <h3>
           <li>Let |document:Document| be [=this=]'s [=relevant global
           object=]'s [=associated Document=].
           </li>
+          <li data-tests="wakelock-active-document.https.window.html">If
```

Comment:
The identical check in lines 333-336 is still present.

Reply:
Moving this to before the permissions policy checks makes sense though.

Reply:
hehe, can't check that fully active enough 🤪